tests/bluechi_test/bluechictl.py

```diff
@@ -168,6 +168,36 @@ def reset_failed(
             expected_result,
         )
 
+    def get_default_target(
+        self,
+        node_name: str = "",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"get-default {node_name}"
+        return self._run(
+            f"GetDefaultTarget of node {node_name}",
+            cmd,
+            check_result,
+            expected_result,
+        )
+
+    def set_default_target(
+        self,
+        node_name: str = "",
+        target: str = "",
+        force: str = "true",
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        cmd = f"set-default {node_name} {target} {force}"
```

Comment:
Force parameter is no longer supported by bluechictl, so please remove it

Reply:
Changed

Reply:
I just want to clarify this part of the code.
Should I only ask for two parameters (node, target) in the `set-default` command?

Reply:
I'd leave the parameter in API (same as systemd), but for bluechictl I wouldn't mention the force parameter and set it to true by default (same as systemctl does)